precovery/frame_db.py

```diff
@@ -235,30 +237,33 @@ def get_frames(
             self.frames.c.obscode,
             self.frames.c.exposure_id,
             self.frames.c.filter,
-            self.frames.c.mjd,
+            self.frames.c.mjd_start,
+            self.frames.c.mjd_mid,
+            self.frames.c.exposure_duration,
             self.frames.c.healpixel,
             self.frames.c.data_uri,
             self.frames.c.data_offset,
             self.frames.c.data_length,
         ).where(
             self.frames.c.obscode == obscode,
             self.frames.c.healpixel == int(healpixel),
-            self.frames.c.mjd >= mjd - 1e-7,
-            self.frames.c.mjd <= mjd + 1e-7,
+            self.frames.c.mjd_mid >= mjd - 1e-7,
+            self.frames.c.mjd_mid <= mjd + 1e-7,
```

Comment:
Should there be support in `get_frames` for using the _exposure_duration_ rather than a static 1e-7 for retrieving frames? Thinking about a case in the future where we may have a precise time and want to have intersecting frames, given that the underlying observations won't necessarily have mjd_mid == mjd.

I realize this isn't the standard use case for `get_frames`, used after propagation to window midpoints.  Perhaps this could be left as-is and what I'm thinking of would be met by a more general function in `frame_db`

Reply:
I struggled with this a little while working on this PR and I think we will have a more concrete answer once we get the per-MJD observations PR completed. Right now this code (I believe) will pull all frames of a single exposure since we end up propagating the orbit to the midpoint MJD and the midpoint MJD will be the same for all frames belonging to a single exposure. The 1e-7 was added to avoid relying on a floating point comparison to extract frames. I'm not sure if adding exposure duration here will have any different impact since the midpoint MJD is the same for all frames that belong to the same exposure?

Reply:
I agree - for precovery as it's working now this is only called in the context of orbits already propagated to the midpoints. 

I did wonder if this would be problematic at all for a case like SDSS, where we will be generating "synthetic" exposures, but again, each of these has a well-defined midpoint time that we would be propagating to. Even if we had exposures overlapping in time, these will have their own associated frames, so I think it's a non-issue